frontend/src/components/dashboard/MuseumGallery.js

```diff
@@ -16,20 +18,33 @@ export default function MuseumGallery(props) {
         displayWidth / artwork.thumbnail.width),
     };
   };
-
+  const getArtworkInfo = (artwork) => {
+    const alt = artwork.thumbnail.alt_text;
+    const linkToImage = getIIIFLevel(artwork, 500);
+    const title = artwork.title;
+    const artworkInfo = new Map();
+    artworkInfo.set('alt', alt);
+    artworkInfo.set('url', linkToImage.url);
+    artworkInfo.set('title', title);
+    return artworkInfo;
+  };
   useEffect(() => {
+    const artworksArr = [];
     fetch(API, {
       headers: {
         'Content-Type': 'application/json',
         'Accept': 'application/json',
       },
     })
         .then((response) => response.json())
-        .then((result) => {
-          document.getElementById('photo')
-              .setAttribute('alt', result.data.thumbnail.alt_text);
-          const linkToImage = getIIIFLevel(result.data, 500);
-          document.getElementById('photo').setAttribute('src', linkToImage.url);
+        .then((artworks) => {
+          for (let i = 0; i < artworks.data.length; i++) {
```

Comment:
There's a `forEach` for javascript arrays in case you didn't know/interested. Could do something like this:
```
.then((artworks) => artworks.filter(checkNull))
.then((artworks) => {
  artworks.forEach( (artwork) => artworksArr.push(getArtworkInfo(artwork));
}
```
Does the same thing though so just a style choice.

Reply:
Great suggestion but I looked into it and I can't call .filter or .foreach since artworks is [a Promise that resolves to a JS obj](https://developer.mozilla.org/en-US/docs/Web/API/Body/json) and not an array :/ I was able to make my code cleaner by using a for in loop (as seen on that website I linked) though!

Reply:
Oh I didn't realize artworks wasn't an array in the json. I did it with the comments array in the json for my portfolio and didn't notice it was different. My bad.